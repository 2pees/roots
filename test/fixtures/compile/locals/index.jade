!!!
html
  head
    title testing
  body
    p= foo
