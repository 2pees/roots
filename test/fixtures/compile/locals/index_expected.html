<!DOCTYPE html><html><head><title>testing</title></head><body><p>bar</p></body></html>